Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      name: tsg_id
+    deprecated: false
+    description: Lists all tags.
+    execution: false
+    name: prisma-sase-tag-list
+    outputs:
+    - contextPath: PrismaSase.Tag.id
+      description: Tag ID.
+      type: String
+    - contextPath: PrismaSase.Tag.name
+      description: Tag name.
+      type: String
+    - contextPath: PrismaSase.Tag.folder
+      description: Tag folder.
+      type: String
+    - contextPath: PrismaSase.Tag.comments
+      description: Tag comments.
+      type: String
+    - contextPath: PrismaSase.Tag.color
+      description: The tag color.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      description: The tag unique name.
+      isArray: false
+      name: name
+      required: true
+    - auto: PREDEFINED
+      default: false
+      description: Tag color.
+      isArray: false
+      name: color
+      predefined:
+      - Red
+      - Green
+      - Blue
+      - Yellow
+      - Copper
+      - Orange
+      - Purple
+      - Gray
+      - Light Green
+      - Cyan
+      - Light Gray
+      - Blue Gray
+      - Lime
+      - Black
+      - Gold
+      - Brown
+      - Olive
+      - Maroon
+      - Red-Orange
+      - Yellow-Orange
+      - Forest Green
+      - Turquoise Blue
+      - Azure Blue
+      - Cerulean Blue
+      - Midnight Blue
+      - Medium Blue
+      - Cobalt Blue
+      - Violet Blue
+      - Blue Violet
+      - Medium Violet
+      - Medium Rose
+      - Lavender
+      - Orchid
+      - Thistle
+      - Peach
+      - Salmon
+      - Magenta
+      - Red Violet
+      - Mahogany
+      - Burnt Sienna
+      - Chestnut
+      required: false
+    - default: false
+      description: Tag comments
+      isArray: false
+      name: comments
+      required: false
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Create a new tag.
+    execution: false
+    name: prisma-sase-tag-create
+    outputs:
+    - contextPath: PrismaSase.Tag.id
+      description: The tag id.
+      type: String
+    - contextPath: PrismaSase.Tag.name
+      description: The tag name.
+      type: String
+    - contextPath: PrismaSase.Tag.folder
+      description: The tag folder.
+      type: String
+    - contextPath: PrismaSase.Tag.color
+      description: The tag color.
+      type: String
+    - contextPath: PrismaSase.Tag.comments
+      description: The tag momments.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or type/name?